extensions/azurecore/package.nls.json

```diff
@@ -20,14 +20,13 @@
 	"config.azureAccountConfigurationSection": "Azure Account Configuration",
 	"config.enablePublicCloudDescription": "Should Azure public cloud integration be enabled",
 	"config.enableUsGovCloudDescription": "Should US Government Azure cloud (Fairfax) integration be enabled",
-	"config.enableUsNatCloudDescription": "Should US National Azure cloud integration be enabled",
 	"config.enableChinaCloudDescription": "Should Azure China integration be enabled",
-	"config.enableGermanyCloudDescription": "Should Azure Germany integration be enabled",
 	"config.azureAuthMethodConfigurationSection": "Azure Authentication Method",
 	"config.azureCodeGrantMethod": "Code Grant Method",
 	"config.azureDeviceCodeMethod": "Device Code Method",
 	"config.noSystemKeychain": "Disable system keychain integration. Credentials will be stored in a flat file in the user's home directory.",
 	"config.piiLogging": "Should Personally Identifiable Information (PII) be logged in the Azure Accounts output channel and the output channel log file.",
 	"config.loggingLevel": "[Optional] The verbosity of logging for the Azure Accounts extension.",
-	"config.authenticationLibrary": "The library used for the AAD auth flow. Please restart ADS after changing this option."
+	"config.authenticationLibrary": "The library used for the AAD auth flow. Please restart ADS after changing this option.",
+	"config.providerSettingsJson": "JSON file path containing custom Azure authentication endpoints. Please restart ADS after changing this option."
```

Comment:
1. This isn't a file path anymore
2. We should generally not be using "ADS" - instead just use the full name
3. Consider changing the last part to something like `Changes to this setting require a restart to take effect.`